src/codec_rav1e.c

```diff
@@ -182,6 +177,13 @@ static avifResult rav1eCodecEncodeImage(avifCodec * codec,
                 goto cleanup;
             }
         }
+        for (uint32_t i = 0; i < codec->csOptions->count; ++i) {
+            avifCodecSpecificOption * entry = &codec->csOptions->entries[i];
+            if (rav1e_config_parse(rav1eConfig, entry->key, entry->value) < 0) {
+                avifDiagnosticsPrintf(codec->diag, "Invalid value for %s: %s.", entry->key, entry->value);
+                return AVIF_RESULT_INVALID_CODEC_SPECIFIC_OPTION;
```

Comment:
No `goto cleanup` necessary? Same for svt.

Reply:
done